test/cpp/microbenchmarks/bm_call_create.cc

```diff
@@ -609,6 +617,125 @@ typedef Fixture<&grpc_server_load_reporting_filter, CHECKS_NOT_LAST>
 BENCHMARK_TEMPLATE(BM_IsolatedFilter, LoadReportingFilter, NoOp);
 BENCHMARK_TEMPLATE(BM_IsolatedFilter, LoadReportingFilter, SendEmptyMetadata);
 
+// Test a filter's start_transport_stream_op_batch in isolation. Fixture 
+// specifies the filter under test (use the Fixture<> template to specify this).
+template <class Fixture>
+static void BM_StartTransportStreamOpBatch(benchmark::State& state) {
```

Comment:
It looks like this duplicates a lot of code from the existing `BM_IsolatedFilter` benchmark.  Consider refactoring to avoid duplication.

Note that this issue may just go away if we switch to the approach of directly calling the filter's methods instead of constructing an entire call stack.

Reply:
Yes, I plan to do refactoring before submitting this PR.

Regarding directly calling the filter's methods vs. constructing an entire call stack, the numbers for this benchmark are so low that we don't think I should invest the time to change it to directly calling the filter's methods.

Reply:
Refactoring in process. 
I deleted BM_IsolatedFilter benchmark and replaced it with BM_CallStackInit. It measures the same thing except I took out the part where it did some random work (either NoOp or SendEmptyMetadata). As we discussed offline, it didn't make sense to measure this random work as part of call creation cost or as part of call_stack_init.